scipy/stats/_multivariate.py

```diff
@@ -5579,10 +5578,18 @@ class uniform_direction_gen(multi_rv_generic):
     :math:`S^2`.
 
     Alternatively, the object may be called (as a function) to fix the `dim`
-    parameter, return a "frozen" `random_direction` random variable:
-
-    >>> rv = uniform_direction(5)
-
+    parameter, return a "frozen" `uniform_direction` distribution. Here
+    we create a ``uniform_direction`` with ``dim=3`` and draw 5 samples.
```

Comment:
```suggestion
    distribution with fixed `dim` parameter. Here,
    we create a `uniform_direction` with ``dim=3`` and draw 5 observations.
```
For consistency throughout stats, I try to use "observation" to refer to a single observation and "sample" to refer to a set of observations. Although I've seen "samples" used interchangeably with "observations" informally, the proposed distinction is consistent with formal definitions I see in the literature, e.g. [here](https://facultystaff.richmond.edu/~rdominey/300/local/ch2.pdf).